src/charactersheet/viewmodels/dm/encounter/index.html

```diff
@@ -1,50 +1,30 @@
-<div class="row">
-  <div class="col-xs-12">
-    <div class="h2">
-      Encounters<br/>
-      <small>Create and manage events and sections of your campaign.</small>
-    </div>
-    <div class="row">
-      <div class="col-xs-12 col-sm-4 col-md-3 tight-right">
-        <div class="panel panel-default">
-          <div class="panel-body tight-inner">
-            <div class="panel-heading">
-              <div class="h4">
-                Encounters
-                <div class="pull-right">
-                  <span class="fa fa-plus fa-lg" style="cursor:pointer;"
-                    data-bind="click: openAddModal"></span>
-                </div>
-              </div>
-            </div>
-            <nested-list params="
-              cells: encounterCells,
-              selectedCell: selectedCell,
-              ondelete: deleteEncounter,
-              onadd: openAddModalWithParent,
-              onselect: selectEncounter"
-            >
-            </nested-list>
-          </div>
-        </div>
-      </div>
-      <div class="col-xs-12 col-sm-8 col-md-9">
-        <!-- ko if: selectedEncounter -->
-        <encounter-detail
-          params="encounter: selectedEncounter"
-        >
-        </encounter-detail>
-        <!-- /ko -->
-        <!-- ko ifnot: selectedEncounter -->
-        <div class="encounter-detail-place-holder well text-center"
-          style="cursor:pointer;" data-bind="click: openAddModal">
-          <p class="lead text-muted heading">Add a new encounter to get started.</p>
-        </div>
-        <!-- /ko -->
-      </div>
+<column-navigation params="context: $data, collapseOn: 'sm'">
+  <!-- ko using: context -->
+  <div class="card-title h4">
+    Encounters<br />
+    <small>
+      Track Events and Places
+    </small>
+    <div class="edit-btn btn-group">
+      <button class="btn btn-link" data-bind="click: openAddModal">
```

Comment:
do we still have modals?

Reply:
I hadn't gotten to the master-list yet. It's removed now. Yes, nested-list still exists.

Reply:
what about the modals, though?